Dockerfile

```diff
@@ -1,5 +1,6 @@
 ARG BASE_IMAGE_TAG="b833677ea06f6409f8555b2743f73bcbd05b6eba"
 FROM ghcr.io/communitiesuk/digital-form-builder-dluhc-runner:$BASE_IMAGE_TAG as base
+
```

Comment:
can we revert this as just whitespace change?

Reply:
will do